src/ba-sak/formateringsvalg.ts

```diff
@@ -88,9 +88,15 @@ export const hentFraDatoValg = (data: IBegrunnelsedata): ValgfeltMuligheter => {
 };
 
 export const hentDuFårEllerHarRettTilValg = (data: IBegrunnelsedata): ValgfeltMuligheter => {
-  if (data.antallBarnOppfyllerTriggereOgHarUtbetaling > 0) {
+  if (
+    data.antallBarnOppfyllerTriggereOgHarUtbetaling > 0 ||
+    data.sokerFaarUtbetaltUtvidetIPerioden
```

Comment:
Gjelder ikke dette for småbarnstillegg også? 
Om ikke hva hvis vi får en annen ytelse som søker kan ha rett til? 
Burde denne gjøres mer generell? Typ `brukerFårMindreEnnRettenTilsier`

Reply:
Nop gjelder ikke for småbarnstillegg! Er derfor jeg har måttet gjøre det på denne måten, og ikke bare sende med totalbeløpet som er utbetalt på søker i perioden. Har avklart med Anna og Birgitte at dette flettefeltet kun er ønskelig at skal ta høyde for utbetaling på barna + utbetaling på søker som er utvidet. Valgfeltet vil aldri bli brukt på begrunnelser som handler om småbarnstillegg. Men føler liksom det er litt hacky å gjøre det på den måten her.

Angående forslaget ditt så trooor jeg ikke det vil bli helt riktig. For her er vi ute etter 0 eller utbetaling, og man kan få mindre enn retten tilsier, men samtidig ha en utbetaling 🤷‍♀️

Reply:
Kunne et mulig navn vært `søkerHarKunRett`?